synthtool/gcp/templates/python_library/noxfile.py.j2

```diff
@@ -193,7 +193,7 @@ def system(session):
         )
 
 {% if samples_test %}
```

Comment:
`samples_test` doesn't appear to be used. Can we remove everything under this code block?

Reply:
Dunno.  All the handwritten clients have separate noxfile(s) for samples these days:  none of them have a `samples` session.  @tswast added that stanza a couple of years back in 8a4579f2276d4814ec2262fc2078f6226da50a63: maybe he can recall if / how it ever got used.

Reply:
I'm not getting any hits on [github search](https://github.com/search?q=org%3Agoogleapis+samples_test&type=code) for `samples_test`. Let's proceed and add repo specific customizations in owlbot.py if we notice any impact in downstream repos.

Reply:
a59fb21